styles.css

```diff
@@ -0,0 +1,96 @@
+body {
+  width: 375px;
+  background-color: #1c1a19;
+  padding: 0 0;
+}
+
+.prinsipal {
+  width: 375px;
+  margin: 0% 0 0 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  font-family: 'Crete Round', 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 400px;
+  height: 92px;
+  background-color: #3c3a39;
+  margin: -10px 0 30px -30px;
+}
```

Comment:
- [ OPTIONAL ] Your navbar appears to have extra unnecessary height. Kindly consider making it shorter to have an attractive and professional UI. You can do this by minimizing the padding on top, as shown on the screenshot below 👇 
  ![portrev2](https://user-images.githubusercontent.com/87394385/199983937-bd3b5ed2-de18-4d8d-a678-d20ab9782725.JPG)